ui/src/androidTest/java/cloudinary/android/ui/CldVideoPlayerInstrumentedTest.java

```diff
@@ -53,8 +53,7 @@ public void testPlayerInitialization() throws InterruptedException {
         assert latchReleased : "CountDownLatch was not released within the timeout.";
         assert cldVideoPlayer[0] != null;
         assert cldVideoPlayer[0].getPlayer() != null;
-        assert cldVideoPlayer[0].getUrl().contains("publicId.m3u8");
-        assert cldVideoPlayer[0].getUrl().contains("sp_auto");
+        assert cldVideoPlayer[0].getUrl().contains("publicId");
```

Comment:
maybe change to `endsWith` or something. Obviously "publicId.m3u8" contains "publicId"

Reply:
We can sense there is also the analytics token
We can disable the analytics token and then try endsWith if it's important

Reply:
so maybe let's just verify that it does not contain `.m3u8`